async_simple/executors/SimpleExecutor.cpp

```diff
@@ -23,15 +23,11 @@ namespace executors {
 static constexpr int64_t kContextMask = 0x40000000;
 
 SimpleExecutor::SimpleExecutor(size_t threadNum) : _pool(threadNum) {
-    [[maybe_unused]] auto ret = _pool.start();
     assert(ret);
```

Comment:
have you tested on debug mode?

Reply:
fix a bug on debug mode, now tests on debug mode and release mode passed.